arangod/Agency/Node.cpp

```diff
@@ -152,19 +164,22 @@ Node::Node(Node const& other)
 /// @brief Must not copy _parent, _store, _ttl
 Node& Node::operator=(VPackSlice const& slice) {
   removeTimeToLive();
-  _children.clear();
-  _value.clear();
+  _children.reset();
+  _value.reset();
   if (slice.isArray()) {
     _isArray = true;
-    _value.resize(slice.length());
+    _value = std::make_unique<std::vector<SmallBuffer>>();
+    _value->reserve(slice.length());
     for (size_t i = 0; i < slice.length(); ++i) {
-      _value.at(i).append(reinterpret_cast<char const*>(slice[i].begin()),
-                          slice[i].byteSize());
+      _value->emplace_back(slice[i].byteSize());
+      memcpy((*_value)[i].data(), slice[i].start(), slice[i].byteSize());
```

Comment:
IMO copying should be performed by a SmallBuffer constructor

Reply:
Changed.